extensions/EXT_texture_compression_bptc/extension.xml

```diff
@@ -75,7 +75,7 @@
     </features>
   </overview>
   <idl xml:space="preserve">
-[NoInterfaceObject]
+[Exposed=Window, LegacyNoInterfaceObject]
```

Comment:
WebGL can be used on workers; throughout this needs to be:
  Exposed=(Window,Worker)
like in the WebGL 1.0 and 2.0 specs.